### ISSUE TITLE
Markdown not rendered in overview

### ISSUE THREAD

**Issue by eNcacz** [mulesoft/api-console#594]
Using this RAML file:
```raml
#%RAML 1.0
title: Test Case
version: 1.0
mediaType: [ application/json ]

/foo:
  get:
    description: |
      Markdown **can** be used in description.

      * it does not work in overview of /foo resource
      * but it works in overview of GET method
    responses:
      200:
        description: everything OK
        body:
          type: object
          description: The response body description
          properties:
            result:
              type: boolean
```
the description is displayed correctly on the GET method description page:
![screenshot_2019-01-31_11-02-12](https://user-images.githubusercontent.com/15668267/52047013-6db59300-2548-11e9-8f19-6aebd5104f2a.png)
but it displayed as plain text (not as markdown) on /foo overview page:
![screenshot_2019-01-31_11-01-46](https://user-images.githubusercontent.com/15668267/52047073-8a51cb00-2548-11e9-83c5-cb34f6816934.png)

**Comment by jarrodek** [comment]
Thank you for reporting. I will fix it soon.

**Comment by jarrodek** [comment]
This is now fixed in 6.0.0
